### PR TITLE
Update BOB Konf video link

### PR DESCRIPTION
**Pull request by fishtreesugar**

### CODE REVIEW

**Comment by bolt12** [comment]
Thanks for the PR! Would you be so kind as to add this one as well: https://www.youtube.com/watch?v=tKIYQgJnGkA&t ? 😄

File: README.md [hunk 1]
```diff
@@ -109,4 +109,4 @@ of issues opened in the `ouroboros-network` repository:
 [io-classes-issues]: https://github.com/input-output-hk/ouroboros-network/issues?q=is%3Aopen+is%3Aissue+label%3Aio-classes
 [contra-tracer]: https://hackage.haskell.org/package/contra-tracer
 [io-sim-por]: https://github.com/input-output-hk/io-sim/blob/main/io-sim/how-to-use-IOSimPOR.md
-[bob-conf]: https://www.youtube.com/watch?v=e74TE0b4xEM&t=13662s
+[bob-conf]: https://youtu.be/uedUGeWN4ZM
```
Comment (bolt12) [inline README.md:112]: It seems already have a link to this talk here: https://github.com/input-output-hk/io-sim/pull/52/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R53

**Review by bolt12 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (coot) [reply README.md:112]: @bolt12 your link goes to this diff which set one on a recursive journey ;).

**Review by bolt12 — APPROVED** [review]

**Comment by bolt12** [comment]
Thank you for your contribution @fishtreesugar !

**Pull request merged**